### PR TITLE
Software 3345

### PR DESCRIPTION
**Pull request by yzz971115**
the change for CampusGrid part

### CODE REVIEW

File: src/webapp/project_reader.py [hunk 1]
```diff
@@ -21,13 +21,20 @@ def get_projects(indir="../projects", strict=False):
...
             continue
+        elif file.endswith("_CAMPUS_GRIDS.yaml"):
+            continue
         project = OrderedDict.fromkeys(["ID", "Name", "Description", "PIName", "Organization", "Department",
                                         "FieldOfScience", "Sponsor"])
         try:
             data = load_yaml_file(os.path.join(indir, file))
+            if 'CampusGrid' in data['Sponsor']:
+                ID = mapping[data['Sponsor']['CampusGrid']['Name']]
+                data['Sponsor']['CampusGrid'].update( {'ID' : ID} )
```
Comment (matyasselmeci) [inline src/webapp/project_reader.py]:
```suggestion
                name = data['Sponsor']['CampusGrid']['Name']
                data['Sponsor']['CampusGrid'] = OrderedDict([("ID", ID), ("Name", name)])
```

File: src/webapp/project_reader.py [hunk 2]
```diff
@@ -38,9 +46,9 @@ def get_projects(indir="../projects", strict=False):
         project.update(data)
         projects.append(project)
 
-    to_output["Projects"]["Project"] = projects
+        to_output["Projects"]["Project"] = projects
```
Comment (matyasselmeci) [inline src/webapp/project_reader.py]:
```suggestion
    to_output["Projects"]["Project"] = projects
```

File: src/webapp/project_reader.py [hunk 3]
```diff
@@ -38,9 +46,9 @@ def get_projects(indir="../projects", strict=False):
...
 
-    return to_output
+        return to_output
```
Comment (matyasselmeci) [inline src/webapp/project_reader.py]:
```suggestion
    return to_output
```

Comment (yzz971115) [reply src/webapp/project_reader.py]: sorry, a careless mistake

**Pull request merged**